CMakeLists.txt

```diff
@@ -74,6 +74,21 @@ endif()
 find_package(Python 3.8 COMPONENTS Interpreter Development.Module REQUIRED)
 find_package(nanobind CONFIG REQUIRED)
 
+# Find jaxlib include directory for XLA FFI headers
+execute_process(
+    COMMAND ${Python_EXECUTABLE} -c "import jaxlib; print(jaxlib.__path__[0])"
+    OUTPUT_VARIABLE JAXLIB_PATH
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    RESULT_VARIABLE JAXLIB_RESULT
+)
```

Comment:
Take a look here for the recommended approach for finding this:

https://github.com/jax-ml/jax/blob/844b2b8b9a5a99f86f1bd96b594c9c2667ff17ae/examples/ffi/CMakeLists.txt#L6-L11